source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/BaseActionElementRenderer.java

```diff
@@ -35,43 +35,6 @@
 
 public abstract class BaseActionElementRenderer implements IBaseActionElementRenderer
 {
-    protected static class ButtonOnLayoutChangedListener implements View.OnLayoutChangeListener
```

Comment:
>ButtonOnLayoutChangedListener [](start = 27, length = 29)

Is removing this a breaking contract change? Is this exposed today as part of the SDK API ? How does this relate to our extensibility story today ?

Reply:
this is not exposed, this listener was in charge of resizing and relocating the image when the button was rendered but it was not needed at all

Reply:
ok thanks the fact tthat this is public threw me off a bit

---
In reply to: [342855406](https://github.com/microsoft/AdaptiveCards/pull/3547#discussion_r342855406) [](ancestors = 342855406)